apps/give-cast/utils/mint.ts

```
@@ -0,0 +1,78 @@
+    while (count < wait.length) {
+      console.log("Retry", count)
+      await new Promise((res) => setTimeout(res, wait[count] * secs))
+      count++
+      try {
+        info = await client.getTransactionReceipt({ hash: txId })
+      } catch (ex) {
+        console.error(ex)
+        continue
+      }
+      console.log("TX", info)
+      if (info?.status == "success") {
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)